dpm/tests/bdd/steps/properties_secrets_step.py

```diff
@@ -130,8 +130,8 @@ def we_get_val(self, result):
 def testing(context):
     counter = 0
     while True:
-        property_value = Env.get_property("USD")
-        secret_value = Env.get_secret("name")
+        property_value = Env().get_instance().get_property("USD")
```

Comment:
Env().get_instance() let's you use the (Inner) Env's methods in a singleton-like way ensuring you get the same instance of env after initialization. 

`Env()` not `Env`